source-hubspot-native/source_hubspot_native/resources.py

```diff
@@ -293,14 +299,13 @@ def open(
         open=open,
         initial_state=ResourceState(
             inc=ResourceState.Incremental(cursor=started_at),
-            backfill=ResourceState.Backfill(next_page=None, cutoff=started_at),
```

Comment:
For this change, if there was no `fetch_page` function, how does the initial `backfill` state matter?

Reply:
Oh, i didnt, i actually forgot to remove that